### ISSUE TITLE
Setting the priority of an attribute to 0 doesn't hide the chart in the study view

### ISSUE THREAD

**Issue by t-pauli** [cBioPortal/cbioportal#8989]
Dear cBioPortal team,

I wish to display certain data in patient view (for instance, birthdate), but I want to avoid displaying the data as chart in the study view. According to this [readme](https://github.com/cBioPortal/cbioportal/blob/master/docs/study-view/priority.md) it is possible, to simply set the priority to 0 in the clinical data files. Sadly, this didn't seem to work for me, as the charts are still being displayed.

Best wishes,
Thomas

**Comment by jjgao** [comment]
@t-pauli thanks for letting us know. We will look into this.

**Comment by inodb** [comment]
@t-pauli What version of the portal did you deploy?

**Comment by alisman** [comment]
@t-pauli a couple possibilties:
Are you logged in to your portal?  Perhaps the chart was added manually and is thus in your session, which would override the priority.  Look in your Add Chart menu for a Reset Charts button.
![image](https://user-images.githubusercontent.com/186521/145233833-d0bade03-b42f-4415-84f8-e45590c07d85.png)
If this is not the case, I would verify that the priority is actually 0 by looking at raw api response of this service: /api/clinical-attributes/fetch, which is a list of the clinical attributes and meta data.

I have tested on our portal that when priority is set to 0, that does indeed hide the chart.  It still appears in add chart menu but in off state.

Let us know what you find.
